core/src/main/scala/org/apache/spark/rapids/tool/benchmarks/BenchmarkArgs.scala

```diff
@@ -0,0 +1,29 @@
+package org.apache.spark.rapids.tool.benchmarks
+
+import org.rogach.scallop.{ScallopConf, ScallopOption}
+
+class BenchmarkArgs(arguments: Seq[String]) extends ScallopConf(arguments) {
+
+    banner("""
+Benchmarker class for running various benchmarks.
+    """)
+
+  val iterations: ScallopOption[Int] = opt[Int](name = "iterations", short = 'i', default = Some(5),
+    descr = "Total iterations to run")
+
+  val warmupIterations: ScallopOption[Int] = opt[Int](name = "warmupIterations", short = 'w' ,
+    default = Some(3), descr = "Number of warmup iterations to run")
+
+  val outputDirectory: ScallopOption[String] = opt[String](name = "outputDirectory", short = 'd',
+    default = Some("."), descr = "Directory to write output to")
+
+  val outputFormat: ScallopOption[String] = opt[String](name = "outputFormat", short = 'o',
+    default = Some("tbl"), descr = "Format of output ( tbl, json)")
```

Comment:
- More details about the options and what each one of them mean.
- Suggest changing "tbl" to "text" 
- the short version 'o'  should be for the outputDirectory. For this option, "'f'" would be more suitable
- QQ why do we specify the name in all the options? shouldn't Scallop do that automatically?

Reply:
* Added more description
* Changed tbl to text. Current PR does not do any processing on output format. Later iteration will merge the JSON output changes
* The short formats have been updated
* The name option is just for verbosity. Saw online implementation using name param hence added it

Reply:
Removed the name param to enable snake-case usage in params